src/pages/collaborations/collaborationManagementForm.jsx

```diff
@@ -3,15 +3,50 @@ import { FormattedMessage, useIntl } from 'react-intl';
 import { get, some } from 'lodash-es';
 import { TextField } from '@material-ui/core';
 import { Autocomplete } from '@material-ui/lab';
+import { useQueryClient } from 'react-query';
+
+import queryKeys from '../../constants/queryKeys';
 import Button from '../../components/Button';
 import CustomAlert from '../../components/Alert';
 import useEstablishCollaborationAsUserManager from '../../models/collaboration/useEstablishCollaborationAsUserManager';
 
+function mutuallyRevoked(members) {
+  //brain too dead right now to make this better
+  const memberKeys = Object.keys(members);
+  const memberViewStates = memberKeys.map(memberKey => {
+    return get(members, [memberKey, 'viewState']);
+  });
```

Comment:
lodash map works on objects. you could do something like 
```
import { map } from `lodash-es`
const memberViewStates = map(members, (member) => get(member, 'viewState'));
```

Reply:
Done.
Very cool about lodash map. ES map's inability to iterate over object keys has been driving me nuts all week.